syntax = "proto3";

package server;

service BackendService {
  rpc ProcessRequest(Request) returns (Response) {}  
  rpc ReportLoad(LoadReport) returns (LoadReportResponse) {} 
}

message Request {
  string data = 1;
}

message Response {
  string result = 1;
}

message LoadReport {
  string address = 1;
  int32 load = 2;
}

message LoadReportResponse {
  bool success = 1;
}
